Formula/mariadb.rb

```diff
@@ -39,14 +34,6 @@ class Mariadb < Formula
     :because => "both install plugins"
 
   def install
-    if build.devel?
```

Comment:
I'm a little confused. I would think this section would now apply to 10.2.6 stable if it applied to 10.2.5 when 10.2.5 was devel.